service/src/main/kotlin/no/nav/su/se/bakover/service/søknad/LukkSøknadRequest.kt

```diff
@@ -0,0 +1,21 @@
+package no.nav.su.se.bakover.service.søknad
+
+import no.nav.su.se.bakover.domain.NavIdentBruker
+import java.time.LocalDate
+import java.util.UUID
+
+sealed class LukkSøknadRequest {
```

Comment:
Jeg har noen ganger før kalt disse for ....Command, random google ting: https://medium.com/@ahmedrizwan/kotlin-design-patterns-the-command-9920b30b2b48

Reply:
Bruker typisk Command når request/respons er forskjellige typer